premake/loong.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index 01eb384..52873af 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -26,3 +26,23 @@ package() {
+   install -Dm755 "bin/release/premake5" "${pkgdir}/usr/bin/premake5"
+   install -Dm644 LICENSE.txt "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+ }
++
++
++# add loongarch64 support
++makedepends=('dos2unix')
++source+=(
++  "premake-riscv.patch::https://github.com/premake/premake-core/commit/82c9d90495940e2d0d574e1c7849e9698f23b090.patch"
```

Comment:
Please also add the reason to apply riscv's patch, although I know  applying it has no negative effects and can avoid rebase conflicts. But you need to add explanation to prevent people from forgetting this.